src/NewCommand.php

```diff
@@ -87,7 +87,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         // TODO it would be better to run not all commands in once, because some of them may fail
-        $process = new Process(implode(' && ', $commands), null, null, null, null);
+        $process = Process::fromShellCommandline(implode(' && ', $commands), null, null, null, null);
```

Comment:
Why the solution is not the same as for code style fixer? I guess also here you could have older symfony installed, so we have to check for the method existence.